{ 'high_step_dorian_bkp': { 'left_arm': [ { 'duration': 2.952,
                                            'name': u'high_step_5',
                                            'positions': [ -1.6698,
                                                           -0.0332,
                                                           -0.6098,
                                                           0.0544,
                                                           -0.0012,
                                                           0.0001,
                                                           0.0],
                                            'starttime': 0.0},
                                          { 'duration': 3.73,
                                            'name': u'high_step_5',
                                            'positions': [ -1.6698,
                                                           -0.0332,
                                                           -0.6098,
                                                           0.0544,
                                                           -0.0012,
                                                           0.0001,
                                                           0.0],
                                            'starttime': 26.183},
                                          { 'duration': 4.458,
                                            'name': u'high_step_7',
                                            'positions': [ -0.17,
                                                           -1.3223,
                                                           -0.11,
                                                           -0.22,
                                                           0.06,
                                                           -0.02,
                                                           0.19],
                                            'starttime': 36.334},
                                          { 'duration': 4.094,
                                            'name': u'high_step_10',
                                            'positions': [ -0.17,
                                                           -0.189,
                                                           -0.11,
                                                           -0.22,
                                                           0.06,
                                                           -0.02,
                                                           0.19],
                                            'starttime': 51.396},
                                          { 'duration': 3.002,
                                            'name': u'high_step_13',
                                            'positions': [ -0.17,
                                                           0.3126,
                                                           -0.11,
                                                           -0.22,
                                                           0.06,
                                                           -0.02,
                                                           0.19],
                                            'starttime': 62.982},
                                          { 'duration': 3.184,
                                            'name': u'high_step_17',
                                            'positions': [ -0.17,
                                                           -0.6953,
                                                           -0.11,
                                                           -0.22,
                                                           0.06,
                                                           -0.02,
                                                           0.19],
                                            'starttime': 77.924}],
                            'left_leg': [ { 'duration': 3.016,
                                            'name': u'high_step_0',
                                            'positions': [ 0.0099,
                                                           -0.0536,
                                                           0.0021,
                                                           -0.0014,
                                                           -0.0063,
                                                           -0.0546],
                                            'starttime': 0.0},
                                          { 'duration': 4.024,
                                            'name': u'high_step_0_left',
                                            'positions': [ -0.0038,
                                                           -0.1993,
                                                           0.0007,
                                                           -0.0005,
                                                           0.0027,
                                                           -0.1973],
                                            'starttime': 3.016},
                                          { 'duration': 2.932,
                                            'name': u'high_step_1_left',
                                            'positions': [ 0.0008,
                                                           -0.1963,
                                                           -0.2525,
                                                           0.5059,
                                                           0.2557,
                                                           -0.2007],
                                            'starttime': 7.04},
                                          { 'duration': 4.192,
                                            'name': u'high_step_2_left',
                                            'positions': [ 0.01,
                                                           -0.3279,
                                                           -1.4502,
                                                           2.4264,
                                                           0.9951,
                                                           -0.22],
                                            'starttime': 9.972},
                                          { 'duration': 2.83,
                                            'name': u'high_step_3_left',
                                            'positions': [ 0.0102,
                                                           -0.3275,
                                                           -1.517,
                                                           1.5889,
                                                           0.1983,
                                                           -0.2201],
                                            'starttime': 14.164},
                                          { 'duration': 2.952,
                                            'name': u'high_step_4_left',
                                            'positions': [ 0.01,
                                                           -0.1648,
                                                           -1.52,
                                                           1.59,
                                                           0.2,
                                                           -0.0629],
                                            'starttime': 16.994},
                                          { 'duration': 3.145000000000003,
                                            'name': u'high_step_4_1_left',
                                            'positions': [ 0.01,
                                                           -0.1649,
                                                           -1.4576,
                                                           1.5538,
                                                           0.1635,
                                                           -0.0629],
                                            'starttime': 19.946},
                                          { 'duration': 3.147,
                                            'name': u'high_step_5_left',
                                            'positions': [ 0.01,
                                                           -0.0392,
                                                           -1.4572,
                                                           1.8067,
                                                           0.528,
                                                           0.0156],
                                            'starttime': 23.091000000000005},
                                          { 'duration': 5.004,
                                            'name': u'high_step_6_left',
                                            'positions': [ 0.01,
                                                           -0.1576,
                                                           -1.46,
                                                           2.0629,
                                                           0.8216,
                                                           0.02],
                                            'starttime': 29.945},
                                          { 'duration': 5.186,
                                            'name': u'high_step_8_left',
                                            'positions': [ 0.01,
                                                           -0.2776,
                                                           -1.46,
                                                           2.0631,
                                                           0.8929,
                                                           0.0985],
                                            'starttime': 40.842},
                                          { 'duration': 3.912,
                                            'name': u'high_step_16_left',
                                            'positions': [ 0.01,
                                                           -0.2748,
                                                           -1.46,
                                                           2.3586,
                                                           1.1815,
                                                           0.1],
                                            'starttime': 74.022}],
                            'neck': [],
                            'right_arm': [ { 'duration': 2.952,
                                             'name': u'high_step_5',
                                             'positions': [ 1.6698,
                                                            0.0332,
                                                            0.6098,
                                                            -0.0544,
                                                            0.0012,
                                                            -0.0001,
                                                            -0.0],
                                             'starttime': 0.0},
                                           { 'duration': 3.73,
                                             'name': u'high_step_5',
                                             'positions': [ 1.6698,
                                                            0.0332,
                                                            0.6098,
                                                            -0.0544,
                                                            0.0012,
                                                            -0.0001,
                                                            -0.0],
                                             'starttime': 26.183},
                                           { 'duration': 4.458,
                                             'name': u'high_step_7',
                                             'positions': [ 0.17,
                                                            1.3223,
                                                            0.11,
                                                            0.22,
                                                            -0.06,
                                                            0.02,
                                                            -0.19],
                                             'starttime': 36.334},
                                           { 'duration': 4.094,
                                             'name': u'high_step_10',
                                             'positions': [ 0.17,
                                                            0.189,
                                                            0.11,
                                                            0.22,
                                                            -0.06,
                                                            0.02,
                                                            -0.19],
                                             'starttime': 51.396},
                                           { 'duration': 3.002,
                                             'name': u'high_step_13',
                                             'positions': [ 0.17,
                                                            -0.3126,
                                                            0.11,
                                                            0.22,
                                                            -0.06,
                                                            0.02,
                                                            -0.19],
                                             'starttime': 62.982},
                                           { 'duration': 3.184,
                                             'name': u'high_step_17',
                                             'positions': [ 0.17,
                                                            0.6953,
                                                            0.11,
                                                            0.22,
                                                            -0.06,
                                                            0.02,
                                                            -0.19],
                                             'starttime': 77.924}],
                            'right_leg': [ { 'duration': 3.016,
                                             'name': u'high_step_0',
                                             'positions': [ -0.0099,
                                                            0.0536,
                                                            -0.0021,
                                                            0.0014,
                                                            0.0063,
                                                            0.0546],
                                             'starttime': 0.0},
                                           { 'duration': 4.002,
                                             'name': u'high_step_0_right',
                                             'positions': [ 0.0225,
                                                            -0.1666,
                                                            -0.0149,
                                                            0.0022,
                                                            0.0094,
                                                            -0.1544],
                                             'starttime': 3.016},
                                           { 'duration': 3.145,
                                             'name': u'high_step_5_right',
                                             'positions': [ 0.02,
                                                            -0.0437,
                                                            -0.1025,
                                                            -0.0467,
                                                            -0.2793,
                                                            0.0478],
                                             'starttime': 23.14},
                                           { 'duration': 5.004,
                                             'name': u'high_step_6_right',
                                             'positions': [ 0.02,
                                                            -0.0337,
                                                            -0.1525,
                                                            0.0301,
                                                            -0.3529,
                                                            0.1286],
                                             'starttime': 29.944},
                                           { 'duration': 5.004,
                                             'name': u'high_step_9_left',
                                             'positions': [ 0.02,
                                                            -0.0237,
                                                            0.4886,
                                                            -2.7079,
                                                            -1.589,
                                                            0.13],
                                             'starttime': 46.422},
                                           { 'duration': 4.094,
                                             'name': u'high_step_10_right',
                                             'positions': [ 0.02,
                                                            -0.0237,
                                                            1.0541,
                                                            -2.7079,
                                                            -1.589,
                                                            0.13],
                                             'starttime': 51.426},
                                           { 'duration': 3.912,
                                             'name': u'high_step_11_right',
                                             'positions': [ 0.02,
                                                            -0.0237,
                                                            1.5474,
                                                            -2.7079,
                                                            -1.589,
                                                            0.13],
                                             'starttime': 55.52},
                                           { 'duration': 3.548,
                                             'name': u'high_step_12_right',
                                             'positions': [ 0.02,
                                                            0.5005,
                                                            1.5474,
                                                            -2.0215,
                                                            -0.2042,
                                                            0.7976],
                                             'starttime': 59.432},
                                           { 'duration': 3.002,
                                             'name': u'high_step_13_right',
                                             'positions': [ 0.02,
                                                            0.5005,
                                                            1.5474,
                                                            -1.2989,
                                                            -0.2042,
                                                            0.7976],
                                             'starttime': 62.980000000000004},
                                           { 'duration': 4.094,
                                             'name': u'high_step_14_right',
                                             'positions': [ 0.02,
                                                            -0.275,
                                                            1.5474,
                                                            -1.0099,
                                                            0.0144,
                                                            0.0515],
                                             'starttime': 65.982},
                                           { 'duration': 3.912,
                                             'name': u'high_step_15_right',
                                             'positions': [ 0.02,
                                                            -0.2436,
                                                            1.1077,
                                                            -0.757,
                                                            0.0873,
                                                            -0.027],
                                             'starttime': 70.076}],
                            'torso': [ { 'duration': 3.03,
                                         'name': u'high_step_0',
                                         'positions': [-0.0, 0.0],
                                         'starttime': 0.0},
                                       { 'duration': 3.184,
                                         'name': u'high_step_5',
                                         'positions': [0.0001, 0.9777],
                                         'starttime': 26.345},
                                       { 'duration': 4.276,
                                         'name': u'high_step_7',
                                         'positions': [0.0, 0.754],
                                         'starttime': 36.455},
                                       { 'duration': 4.094,
                                         'name': u'high_step_10',
                                         'positions': [0.0, 0.3102],
                                         'starttime': 51.457},
                                       { 'duration': 3.002,
                                         'name': u'high_step_13',
                                         'positions': [0.0, -0.0983],
                                         'starttime': 63.022}]}}
